src/sonic-config-engine/sonic-cfggen

```diff
@@ -364,6 +364,15 @@ def main():
         if ports is None:
             print('Failed to get port config', file=sys.stderr)
             sys.exit(1)
+        for port in ports.values():
```

Comment:
If port['pfc_asym'] = 'off' can be set, do we need this change?

Reply:
Need something to represent null value (here it is '-'), and in null value case, need to delete pfc_asym entry for those ports.